code/src/components/SendThought.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import ButtonSubmit from './ButtonSubmit'
+import '../styles/SendThought.css'
+
+const SendThought = ({
+  handleFormSubmit,
+  newThought,
+  onNewThoughtChange
+}) => {
+  return (
+  <div className="thoughts-container-whitesmoke">
+    <form onSubmit={handleFormSubmit} className="thoughts-container">
+    <label htmlFor="newThought" className="title-question">What's making you happy right now?</label>
+      <textarea
+        className="user-input-textarea"
+        id="newThought"
+        type="text"
+        maxLength="140"
+        value={newThought}
+        onChange={onNewThoughtChange}
+        placeholder="Write your thoughts here, you will contribute to our database 😊">
+      </textarea>
+      <p
+        className={`counter-for-text-input ${newThought.length > 130 ? "red" : "black"}`}
```

Comment:
I like this way of conditionally applying styling! It is concise and easy to read! One suggestion might be, just like my comment above, use other naming for the classes - maybe something like "default" and "warning" ... or something!